pages/index.js

```diff
@@ -1,3 +1,25 @@
-export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+import { fakeFetcher } from "../helpers/api";
+import useSWR from "swr";
+const apiUrl = "/api/data";
```

Comment:
why is this static string a variable, instead of inlined?

using the static string directly as an inline argument is more idiomatic.